pelican/themes/simple/templates/base.html

```diff
@@ -32,11 +33,11 @@
         {% endblock head %}
 </head>
 
-<body id="index" class="home">
-        <header id="banner" class="body">
+<body>
+        <header>
                 <h1><a href="{{ SITEURL }}/">{{ SITENAME }}{% if SITESUBTITLE %} <strong>{{ SITESUBTITLE }}</strong>{% endif %}</a></h1>
-        </header><!-- /#banner -->
-        <nav id="menu"><ul>
+        </header>
+        <nav><ul>
```

Comment:
Changes like this one will break websites that extend the `Simple` theme; what was the motivation?